californium-core/src/main/java/org/eclipse/californium/core/network/CoapEndpoint.java

```diff
@@ -217,9 +217,18 @@
 	/** Parser to convert datagrams to messages. */
 	private final DataParser parser;
 
+	/** A store containing data about message exchanges. */
+	private final MessageExchangeStore exchangeStore;
+
+	/** A Store containing data about observation. */
+	private ObservationStore observationStore;
```

Comment:
`final exchangeStore` and non-final `observationStore`, is that intended?

Reply:
nope, I will fix it by adding final keyword